app/views/users/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<% @active_menu = "users" %>
+
+<h1>Users</h1>
+<div class="row">
+  <div class="col-md-8">
+    <table class="table table-hover users">
+      <thead>
+        <tr>
+          <th>Email</th>
+        </tr>
+      </thead>
+      <tbody>
+        <%= render @users %>
+        <tr class="<%= "hidden" if @users.any? %>" data-role="empty-table">
```

Comment:
I don't think this special case needs to be here, since we don't delete users. So we always will have at least one user.